addons/bus/tests/test_assetsbundle.py

```diff
@@ -32,7 +32,8 @@ def patched_sendone(self, channel, message):
             """
             Control API and number of messages posted to the bus
             """
-            sendones.append((channel, message))
+            if channel[1] == 'bundle_changed':
+                sendones.append((channel, message))
```

Comment:
Looks safe considering how it used in context of test, I guess.

Reply:
It's strange that channel_pin (if that is the broadcast that makes an issue) is called during this tour. Probably there should be a way to properly set pin state in demo that so that this RPC does not happen at all.

Reply:
It seems that it's adding a new channel for the "bundle_changed" thing, I suppose that's what triggers the channel_pin

Reply:
`'14.0-mail-rejoin-chat-xdu-test', 'res.partner', 3)`
```
{
  'id': 1,
  'name': 'general',
  'uuid': '43758522-0fbb-483e-a256-7b44a1e4c338',
  'state': 'open',
  'is_minimized': False,
  'channel_type': 'channel',
  'public': 'groups',
  'mass_mailing': False,
  'moderation': False,
  'is_moderator': False,
  'group_based_subscription': True,
  'create_uid': 1,
  'last_message_id': 9,
  'message_needaction_counter': 0,
  'message_unread_counter': 5,
  'seen_message_id': False,
  'custom_channel_name': False,
  'is_pinned': False,
  'members': [{
    'id': 2,
    'name': 'OdooBot',
    'email': 'odoobot@example.com'
  }, {
    'id': 7,
    'name': 'Marc Demo',
    'email': 'mark.brown23@example.com'
  }, {
    'id': 3,
    'name': 'Mitchell Admin',
    'email': 'admin@yourcompany.example.com'
  }]
}
```

Reply:
Do you know what part of the code is triggering it? Which RPC/called where from the JS?

payload lacks contextual information (unless `info` is set but that's not the case here).

Reply:
What I think is that the channel general (from demo data) should be pinned by default for all users that are part of it (which should also be done in demo data).

This is the "problem" of tours & tests relying on demo data, is that they don't work on a "clean" database so we have such side effects.

Reply:
> Do you know what part of the code is triggering it? Which RPC/called where from the JS?

Not really, probably the init_messaging

Reply:
As per discussed, but just to be clear I'm expecting this to be changed so I'll mark change request on PR.